packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -768,6 +792,8 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
         bindChannel: (channel: IFluidDataStoreChannel) => void,
         private readonly snapshotTree: ISnapshotTree | undefined,
         protected readonly isRootDataStore: boolean,
+        disableIsolatedChannels: boolean,
+        protected readonly summaryFormatVersion: DataStoreSummaryFormatVersion,
```

Comment:
Why is `summaryFormatVersion` needed for creation. When writing the version to attributes blob, it uses `disableIsolatedChannels` to generate summary version. The only place this is used is when getting initial snapshot. Can't we get it from `disableIsolatedChannels` there too?

Reply:
I will add comments to explain this, but `disableIsolatedChannels` instructs how to _write_ summaries. The `summaryFormatVersion` which we parse from .metadata/.component blobs instructs how to _read_ the current base snapshot.